### ISSUE TITLE
show "comments" columns in all references tabs

### ISSUE THREAD

**Issue by d2k2-git** [x64dbg/x64dbg#3466]
### Feature type

Quality of life

### Describe the feature

Hello x64dbg lovers,

i really enjoy to use this tool. also for code analysis because its lightweight and fast compared to IDA. I have often projects which are automatically labeled and commented by generated scripts of my tools.

i can see currently the comments when i open the "comments" tab or when i browse the code in the CPU tab. however, it would be also nice to see the comments in every tab view under "references". 

Address | Disassmembly | Comment

![image](https://github.com/user-attachments/assets/2811907c-c2e9-4a6d-9c04-fb40ed46a0a5)

**Comment by mrexodia** [comment]
I agree this would be nice to have, but the way that the references are currently implemented this is going to require updates in all the places that emit them.

The current design is that the references are a generic table that is populated by the backend/plugins with headers and data set manually. It would require a modification of the API to say 'this is referencing an instruction' and then fetch the disassembly/comments/labels/etc automatically in the GUI when needed.
